src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$font: 'Roboto', sans-serif;
+$main-color: #060b35;
+$secondColor: #616070;
+$fontWeightStandard: 400;
```

Comment:
Variables should be named using kebab-case according to the code style rules. Please rename `$secondColor` to `$second-color` and `$fontWeightStandard` to `$font-weight-standard` to maintain consistency.